packages/components/src/components/ui-shell/_side-nav.scss

```diff
@@ -344,6 +344,15 @@
     height: mini-units(6);
   }
 
+  //----------------------------------------------------------------------------
+  // Side-nav > Navigation > Divider
+  //----------------------------------------------------------------------------
+  .#{$prefix}--side-nav__divider {
+    height: 1px;
+    margin: $spacing-03 $spacing-05;
+    background-color: $ui-03;
```

Comment:
So we don't use the component color tokens in the UI Shell right now (hopefully correcting this in the future). We actually just use the language colors and it will look the same in each theme. Call the `Gray20` element token instead here.

Reply:
Ah, right, good catch!